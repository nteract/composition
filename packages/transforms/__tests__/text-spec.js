import React from "react";

import { mount } from "enzyme";

import Text from "../src/text";

describe("Text", () => {
  it("renders plain text", () => {
    const wrapper = mount(<Text data={"hey"} />);
    expect(wrapper.html()).toEqual("<code><span>hey</span></code>");

    const component = wrapper.instance();

    expect(component.shouldComponentUpdate()).toEqual(true);
  });
});
