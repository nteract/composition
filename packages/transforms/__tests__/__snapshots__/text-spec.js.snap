// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders plain text 1`] = `
<code>
  <span
    style={Object {}}
  >
    hey
  </span>
</code>
`;
